src/rpc/rawtransaction.cpp

```diff
@@ -185,9 +231,9 @@ static UniValue getrawtransaction(const JSONRPCRequest& request)
     }
 
     // Accept either a bool (true) or a num (>=1) to indicate verbose output.
-    bool fVerbose = false;
+    int fVerbose = 0;
```

Comment:
verbosityLevel would be more accurate

Reply:
Original implementation was capable to use false, 0  or true any number expect 0.
Current change give option 2 but rest states the same.
I expected that this should be backward compatible.